Purchases/Public/Purchases.swift

```diff
@@ -651,24 +615,26 @@ public extension Purchases {
      * This function will alias two appUserIDs together.
      *
      * - Parameter alias: The new appUserID that should be linked to the currently identified appUserID
-     * - Parameter maybeCompletionBlock: An optional completion block called when the aliasing has been successful.
+     * - Parameter completion: An optional completion block called when the aliasing has been successful.
      * This completion block will receive an error if there's been one.
      */
-    @objc func createAlias(_ alias: String, completionBlock maybeCompletionBlock: ReceivePurchaserInfoBlock?) {
+    @available(*, deprecated, message: "use logIn instead")
+    @objc(createAlias:completion:)
```

Comment:
I think if we use the `renamed` attribute in `@available`, xcode auto-suggests the fix-it, which would be cool

Reply:
💯